library/network_connections.py

```diff
@@ -1911,7 +1933,12 @@ def run_prepare(self):
 
             name = connection["name"]
             if not name:
-                assert connection["persistent_state"] == "absent"
+                if not connection["persistent_state"] == "absent":
```

Comment:
Just from my curiosity..., is
`if not connection["persistent_state"] == "absent":`
equivlalent to
`if connection["persistent_state"] != "absent":`
?

Reply:
> Just from my curiosity..., is
> `if not connection["persistent_state"] == "absent":`
> equivlalent to
> `if connection["persistent_state"] != "absent":`
> ?

Yes